### ISSUE TITLE
NJsonSchema > 9.11.0 causes test failures

### ISSUE THREAD

**Issue by qdot** [buttplugio/buttplug-csharp#537]
If NJsonSchema is upgraded beyond 9.11.0, we get test failures in the Buttplug.Client.IPCConnector.Test and Buttplug.Client.WebsocketConnector.Test TestDeviceMessageHelper functions, and in Buttplug.Core.Test TestRotateCmd. This is most likely because these are the only places to exercise generic messages in the schema validator.

Seems to have something to do with subcommands in generic messages.

```
Message: Buttplug.Core.ButtplugMessageException : Message does not conform to schema: ArrayItemNotValid: #/[0]
{
  PropertyRequired: #/[0].RotateCmd.Rotations.Index
  PropertyRequired: #/[0].RotateCmd.Rotations.Speed
  PropertyRequired: #/[0].RotateCmd.Rotations.Clockwise
}
```

```
Message: Buttplug.Core.ButtplugMessageException : Message does not conform to schema: ArrayItemNotValid: #/[0]
{
  PropertyRequired: #/[0].VibrateCmd.Speeds.Index
  PropertyRequired: #/[0].VibrateCmd.Speeds.Speed
}
```

**Comment by qdot** [comment]
This is actually a schema problem. See buttplugio/buttplug-schema#10

Leaving this bug here because once this is fixed, it will need to be brought into this repo.

**Comment by qdot** [comment]
0.9.12 upgrade along with schema fix landed in 0.3.1. Closing.
